superbench/benchmarks/micro_benchmarks/disk_performance.py

```diff
@@ -190,25 +190,25 @@ def _process_raw_result(self, cmd_idx, raw_output):
             fio_output = json.loads(raw_output)
 
             jobname = fio_output['jobs'][0]['jobname']
```

Comment:
Is it a bug? It's like a revision for the result summary.

Reply:
It is, because current metric name uses reserved character ':', which might cause metric post processing failure.

Reply:
Yes. Therefore, it's not a bug. It's the revision to support metric post processing.

Reply:
Shall we split this into 2 PRs?

Reply:
You can revise your title directly.

Reply:
Fixed.